templates/service-template.yml

```diff
@@ -285,6 +285,36 @@ parameters:
   description: A list of users that are allowed to create standard KAFKA instances via their organisation in a yaml format.
   value: "[{id: 13640203, max_allowed_instances: 5, any_user: true, registered_users: []}, {id: 12147054, max_allowed_instances: 1, any_user: true, registered_users: []}, {id: 13639843, max_allowed_instances: 1, any_user: true, registered_users: []}]"
 
+- name: TENNANT_USERNAME
+  displayName: Username key for cluser manager.
+  description: Username key for cluser manager.
```

Comment:
The description here and the ones below are missing the information that this value is the `"token claims key"` and it will be used to retrieve the corresponding username/orgId etc

Reply:
I think the same applies to the parameter name.